hazelcast-sql-core/src/test/java/com/hazelcast/sql/misc/SqlNoDeserializationTest.java

```diff
@@ -207,8 +208,17 @@ private void populate(boolean fail) {
             localMap.put(new PersonKey(i, fail), new Person(fail));
         }
 
```

Comment:
could you please add a short explanatory doc to this `populate` method?

Reply:
The method is simplified as explained in the next comment.